less/home-page.less

```diff
@@ -57,19 +86,46 @@
         margin-bottom: 26px;
       }
 
-      .btn {
+      button {
+        .custombtn;
+      }
+
+      @media @phone {
+        width: 100%;
       }
     }
 
     .walk-img {
       width: 46%;
       margin-left: 4%;
-      
+      position: relative;
+
       img {
         width: 100%;
         height: auto;
         border-radius: 10px;
       }
+
+      @media @phone {
+        width: 100%;
+        margin: 0 auto;
+        padding: 20px 0;
+      }
+
+      .orbit {              //stretch goal animation
```

Comment:
great usage of positioning and using top/left/right/bottom to both position and size the element!